src/components/customCharts/SimpleBarChart.tsx

```diff
@@ -26,21 +30,26 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const CHART_PADDING = 10;
+const CHART_DETAILS_COLOR = 'rgb(72,72,72)';
 
-const handleElementClick = (setClickedCallback: any, objects: any, clickPositions: any) => {
+const handleElementClick = (
+  setClickedCallback: any,
+  objects: any,
+  clickPositions: any
```

Comment:
A to nie będzie czymś w stylu " HTMLCanvasElement" lub "React.MouseEvent<MouseEvent>"? Też pytanie bardziej na przyszłość, nie na to rev :)